commands/chain_start.go

```diff
@@ -0,0 +1,67 @@
+package commands
+
+import (
+	"github.com/cybercongress/cybernode/common"
+	"github.com/spf13/cobra"
+	"github.com/docker/docker/api/types"
+	"log"
+	"io"
+	"os"
+	"github.com/docker/docker/api/types/container"
+	"fmt"
+	"golang.org/x/net/context"
+	"strings"
+)
+
+func startNodeCmd(chain common.Chain) *cobra.Command {
+	return &cobra.Command{
+		Use: "start",
+		Run: func(cmd *cobra.Command, args []string) {
+
+			ctx := context.Background()
+
+			pullOrUpdateImage(ctx, chain)
+
+			dockerContainer, err := dockerClient.ContainerInspect(ctx, chain.DockerContainerName())
+			if err != nil {
+				createContainer(chain, ctx)
+			} else if dockerContainer.State.Running || dockerContainer.State.Restarting {
+				log.Fatal(strings.Title(chain.Name), " already running")
```

Comment:
is there any option not to use log.Fatal? or this  is standard practice?
else log.Fatal prints output to std.output not std.error